src/ui/src/sessioninfo.cpp

```diff
@@ -34,7 +34,8 @@ void SessionInfo::retrieveFromStorage( QSettings& settings )
 
     if ( settings.value( "version", 0 ).toInt() == SESSION_VERSION ) {
         windows_.clear();
-        const auto windowsCount = settings.beginReadArray( "windows" );
+        const int windowsCount = settings.beginReadArray( "windows" );
```

Comment:
Another settings place. Let's remove reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3